assets/constants.ts

```diff
@@ -5054,132 +5054,132 @@ export default {
     "direction": "ltr"
   }, {
     "id": "ak",
-    "description": "Akan",
+    "description": "Ákán (Akan)",
     "relatedLanguages": ["ak"],
     "direction": "ltr"
   }, {
     "id": "sq",
-    "description": "Albanian",
+    "description": "shqip (Albanian)",
     "relatedLanguages": ["sq"],
     "direction": "ltr"
   }, {
     "id": "ar",
-    "description": "Arabic",
+    "description": "العربية (Arabic)",
     "relatedLanguages": ["ar"],
     "direction": "rtl"
   }, {
     "id": "az",
-    "description": "Azerbaijani",
+    "description": "Azeri (Azerbaijani)",
     "relatedLanguages": ["az"],
     "direction": "ltr"
   }, {
     "id": "bg",
-    "description": "Bulgarian",
+    "description": "български (Bulgarian)",
     "relatedLanguages": ["bg"],
     "direction": "ltr"
   }, {
     "id": "bn",
-    "description": "Bangla",
+    "description": "বাংলা (Bangla)",
     "relatedLanguages": ["bn"],
     "direction": "ltr"
   }, {
     "id": "ms",
-    "description": "Bahasa Melayu",
+    "description": "بهاس ملايو(Bahasa Melayu)",
     "relatedLanguages": ["ms"],
     "direction": "ltr"
   }, {
     "id": "ca",
-    "description": "Catalan",
+    "description": "català (Catalan)",
     "relatedLanguages": ["ca"],
     "direction": "ltr"
   }, {
     "id": "zh",
-    "description": "Chinese",
+    "description": "中文 (Chinese)",
     "relatedLanguages": ["zh"],
     "direction": "ltr"
   }, {
     "id": "hr",
-    "description": "Croatian",
+    "description": "hrvatski (Croatian)",
     "relatedLanguages": ["hr"],
     "direction": "ltr"
   }, {
     "id": "cs",
-    "description": "Czech",
+    "description": "čeština (Czech)",
     "relatedLanguages": ["cs"],
     "direction": "ltr"
   }, {
     "id": "da",
-    "description": "Danish",
+    "description": "dansk (Danish)",
     "relatedLanguages": ["da"],
     "direction": "ltr"
   }, {
     "id": "prs",
-    "description": "Dari",
+    "description": "دری (Dari)",
     "relatedLanguages": ["prs"],
     "direction": "rtl"
   }, {
     "id": "nl",
-    "description": "Dutch",
+    "description": "Nederlands (Dutch)",
     "relatedLanguages": ["nl"],
     "direction": "ltr"
   }, {
     "id": "ee",
-    "description": "Ewe",
+    "description": "Eʋegbe",
```

Comment:
Why does this language not get a parenthesis?

Reply:
Oh sorry, missed adding the English form of the language. Thank you!! Changed.